sigstore/transparency.py

```diff
@@ -24,9 +24,9 @@
     BaseModel,
     ConfigDict,
     Field,
-    FieldValidationInfo,
     StrictInt,
     StrictStr,
+    ValidationInfo,
```

Comment:
I haven't root caused this, but this looks like a semver-breaking change in Pydantic.

Reply:
Hmm, more borkage with Pydantic: https://github.com/sigstore-conformance/extremely-dangerous-public-oidc-beacon/actions/runs/6315666569/job/17148672809

Reply:
https://github.com/sigstore-conformance/extremely-dangerous-public-oidc-beacon/pull/6

Reply:
> I haven't root caused this, but this looks like a semver-breaking change in Pydantic.

Either that or I used supposedly non-public API by accident... https://github.com/pydantic/pydantic/pull/7542 happened last friday

Reply:
Ah yep, could be. It looks like `FieldValidationInfo` might have been deprecated at some point (prior to removal), but I can't find a good reference for that (or for why).